src/styles/home.css

```diff
@@ -66,7 +66,15 @@ nav ul li {
 }
 
 .login {
-    color: var(--black);
+    color: var(--white);
+}
```

Comment:
Use of var for color representation is good. Use of rem unit is consistently done.